xarray/tests/temp/cftime_resample_pandas_comparison.py

```diff
@@ -0,0 +1,118 @@
+ps = pd.Series(np.arange(100, 100+ti.size), index=ti)
+print(ps.resample('3M', label='left', closed='left').max())
+print(ps.resample('3M', label='right', closed='left').max())
+print(ps.resample('3M', label='left', closed='right').max())
+print(ps.resample('3M', label='right', closed='right').max())
+print(ps.resample('3MS', label='left', closed='left').max())
+print(ps.resample('3MS', label='right', closed='left').max())
+print(ps.resample('3MS', label='left', closed='right').max())
+print(ps.resample('3MS', label='right', closed='right').max())
+print(ps.resample('3M', label='left', closed='left').mean())
+print(ps.resample('3M', label='right', closed='left').mean())
+print(ps.resample('3M', label='left', closed='right').mean())
+print(ps.resample('3M', label='right', closed='right').mean())
+print(ps.resample('3MS', label='left', closed='left').mean())
+print(ps.resample('3MS', label='right', closed='left').mean())
+print(ps.resample('3MS', label='left', closed='right').mean())
+print(ps.resample('3MS', label='right', closed='right').mean())
+print(ps.resample('2M', label='left', closed='left').mean())
+print(ps.resample('2M', label='right', closed='left').mean())
+print(ps.resample('2M', label='left', closed='right').mean())
+print(ps.resample('2M', label='right', closed='right').mean())
+print(ps.resample('2MS', label='left', closed='left').mean())
+print(ps.resample('2MS', label='right', closed='left').mean())
+print(ps.resample('2MS', label='left', closed='right').mean())
+print(ps.resample('2MS', label='right', closed='right').mean())
+# Checking how label and closed args affect outputs
+ti = pd.date_range('2000-01-01', periods=9, freq='T', tz='UTC')
+ps = pd.Series(np.arange(100, 100+ti.size), index=ti)
+print(ps.resample('3T', label='left', closed='left').mean())
+print(ps.resample('3T', label='right', closed='left').mean())
+print(ps.resample('3T', label='left', closed='right').mean())
+print(ps.resample('3T', label='right', closed='right').mean())
+ti = pd.date_range('2000', periods=30, freq='MS')
+ps = pd.Series(np.arange(100, 100+ti.size), index=ti)
+print(ps.resample('6MS', label='left', closed='left').max())
+print(ps.resample('6MS', label='right', closed='left').max())
+print(ps.resample('6MS', label='left', closed='right').max())
+print(ps.resample('6MS', label='right', closed='right').max())
+# Checking different aggregation funcs, also checking cases when label and closed == None
+ti = pd.date_range('2000', periods=30, freq='MS')
+ps = pd.Series(np.arange(100, 100+ti.size), index=ti)
+print(ps.resample('MS').mean())  # essentially doing no resampling, should return original data
+print(ps.resample('6MS').mean())
+print(ps.resample('6MS').asfreq())  # results do not match since xarray makes asfreq = mean (see resample.py)
+print(ps.resample('6MS').sum())
+print(ps.resample('6MS').min())
+print(ps.resample('6MS').max())
+
+
+# Upsampling comparisons:
+# At seconds-resolution, xr.cftime_range is 1 second off from pd.date_range
```

Comment:
It would be great if you could open a separate issue for this with a minimal working example; this was not intentional and I think we would consider it a bug.